WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/FIle List/ProductDownloadListViewController.swift

```diff
@@ -0,0 +1,208 @@
+        onCompletion = completion
+        super.init(nibName: type(of: self).nibName, bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureAddButton()
+        configureAddButtonSeparator()
+        registerTableViewCells()
+        configureTableView()
+        configureNavigationBar()
+        handleSwipeBackGesture()
+    }
+}
+
+// MARK: - View Configuration
+//
+private extension ProductDownloadListViewController {
+    func registerTableViewCells() {
+        tableView.register(ImageAndTitleAndTextTableViewCell.loadNib(), forCellReuseIdentifier: ImageAndTitleAndTextTableViewCell.reuseIdentifier)
+    }
+
+    func configureAddButton() {
+        addButton.setTitle(NSLocalizedString("Add File", comment: "Action to add downloadable file on the Product Downloadable Files screen"), for: .normal)
+        addButton.addTarget(self, action: #selector(addButtonTapped), for: .touchUpInside)
+        addButton.applySecondaryButtonStyle()
+    }
+
+    func configureAddButtonSeparator() {
+        addButtonSeparator.backgroundColor = .systemColor(.separator)
+    }
+
+    func configureTableView() {
+        view.backgroundColor = .listBackground
+        tableView.backgroundColor = .listBackground
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.removeLastCellSeparator()
+        tableView.dragInteractionEnabled = true
+        tableView.dragDelegate = self
+        tableView.dropDelegate = self
+        tableView.reloadData()
+    }
+
+    func configureNavigationBar() {
+        configureTitle()
+        configureRightButtons()
+    }
+
+    func configureTitle() {
+        title = NSLocalizedString("Downloadable Files",
+                                  comment: "Edit product downloadable files screen - Screen title")
+    }
+
+    func configureRightButtons() {
+        var rightBarButtonItems = [UIBarButtonItem]()
+
+        let downloadSettingsBarButton: UIBarButtonItem = {
+            let button = UIBarButtonItem(image: .moreImage,
+                                         style: .plain,
+                                         target: self,
+                                         action: #selector(downloadSettingsButtonTapped))
+            button.accessibilityTraits = .button
+            button.accessibilityLabel = NSLocalizedString("View downloadable file settings",
+                                                          comment: "The action to update downloadable files settings for a product")
+            return button
+        }()
+        rightBarButtonItems.append(downloadSettingsBarButton)
+
+        let doneButtonTitle = NSLocalizedString("Done",
+                                                comment: """
+Edit product downloadable files screen - button title to apply changes to downloadable files selection
+""")
```

Comment:
Same issue with triple quotes here. 🙂 Please see my comment [above](https://github.com/woocommerce/woocommerce-ios/pull/2814/files#r492363985) .

Reply:
Sorry, I wasn't aware of that. I will remove those and make sure not to use it again.  🙂